support/client/lib/vwf/model/mil-sym.js

```diff
@@ -700,10 +707,10 @@ define( [ "module",
                 symbolCode = cws.addAffiliationToSymbolId( node.symbolID, node.affiliation );
             }
             
-            var img = rendererMP.RenderSymbol2D(node.ID,node.fullName,node.description, symbolCode, controlPts, node.width, node.height, null, node.modifiers, format);
+            var img = rendererMP.RenderSymbol2D(node.ID,node.fullName,node.description, symbolCode, controlPts, node.bounds[0], node.bounds[1], null, node.modifiers, format);
 
             if ( !!img && !!img.image ) {
```

Comment:
I've become a fan of @davideaster 's way of chaining non-null checks like this:

    if ( ( img || {} ).image ) {

... I offer it in case you find it interesting.  :smile: